hardhat/scripts/setup-testnet.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-cli/main/scripts/install.sh | sh -s
+export PATH=~/bin:$PATH
```

Comment:
Hmmm, can we maybe create a local tmp folder here (and git ignore it) to download and execute? I'd rather not modify the dev's path

Reply:
Or, ideally we run this whole thing in a container and just expose the rpc url?